### PR TITLE
Rename _pixelbuf to adafruit_pixelbuf, alias.

### PR DESCRIPTION
**Pull request by kattni**
- Renames `_pixelbuf` to `adafruit_pixelbuf`.
- Aliases `_pixelbuf` to `adafruit_pixelbuf` for backwards compatibility.

### CODE REVIEW

**Comment by kattni** [comment]
The updates to NeoPixel and DotStar are incoming.

File: shared-bindings/adafruit_pixelbuf/__init__.c [hunk 1]
```diff
@@ -29,20 +29,22 @@
...
+#include "shared-bindings/adafruit_pixelbuf/PixelBuf.h"
 
 
 //| """A fast RGB(W) pixel buffer library for like NeoPixel and DotStar
 //|
-//| The `_pixelbuf` module provides the :py:class:`PixelBuf` class to accelerate
+//| The `adafruit_pixelbuf` module provides the :py:class:`PixelBuf` class to accelerate
 //| RGB(W) strip/matrix manipulation, such as DotStar and Neopixel.
 //|
+//| Also available as `_pixelbuf`. This usage has been deprecated.
```
Comment (dhalbert) [inline shared-bindings/adafruit_pixelbuf/__init__.c:41]: `_pixelbuf` is no longer an auto-generated module name. The easiest thing to do here is probably to just do:
```suggestion
//| Also available as ``_pixelbuf``. This usage has been deprecated.
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Looks good! Some things to change.

**Comment by dhalbert** [comment]
Build failures are due to xtensa build caching. Nothing to do with your changes. To fix, we need to change the cache key:
Change
`.github/workflows/build.yml`, line 534:
OLD
```
        key: ${{ runner.os }}-idf-tools-${{ hashFiles('.git/modules/ports/esp32s2/esp-idf/HEAD') }}-20210506
```
NEW
```
        key: ${{ runner.os }}-idf-tools-${{ hashFiles('.git/modules/ports/esp32s2/esp-idf/HEAD') }}-20210716
```
In general, we update the key to today's date.

It is not clear to me at all why and when this happens, but this kind of thing is the fix.

**Review by dhalbert — APPROVED** [review]
Great!

**Pull request merged**